<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LowPassDialog</class>
 <widget class="QDialog" name="LowPassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Matrix:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="5">
      <widget class="QLineEdit" name="lineEdit33">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit11">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit21">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit31">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="lineEdit23">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit22">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit13">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit12">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEdit32">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radiobtn_spatial">
       <property name="text">
        <string>时域卷积</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobtn_fre">
       <property name="text">
        <string>频域点乘</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LowPassDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LowPassDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
